libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/location/NavigationLocationProvider.kt

```diff
@@ -172,8 +170,9 @@ class NavigationLocationProvider : LocationProvider {
     ): (ValueAnimator.() -> Unit) {
         val evaluator = PuckAnimationEvaluator(keyPoints)
         return {
-            clientOptions?.also { apply(it) }
+            duration = LocationComponentConstants.DEFAULT_INTERVAL_MILLIS
```

Comment:
Let's add a `TODO` comment linking to the Maps ticket so we remove this from our side when they fix it.

Reply:
cc @mapbox/maps-android

Reply:
Done.